app/src/main/java/org/fossasia/openevent/data/Session.java

```diff
@@ -142,42 +151,25 @@ public void setTrack(int track) {
         this.track = track;
     }
 
-    public String getSpeakers() {
-        return speakers;
-    }
-
-    public void setSpeakers(String speakers) {
-        this.speakers = speakers;
-    }
-
-
-    public int[] getSpeakersAsIntArray() {
-        return new int[]{1, 2, 3};
-        //TODO: Parse and deserialise properly
-    }
-
 
     public String getLevel() {
-        return level;
+        return escapeChar(level);
     }
 
     public void setLevel(String level) {
         this.level = level;
     }
 
-    public int getMicrolocation() {
-        return microlocation;
-    }
-
-    public void setMicrolocation(int microlocation) {
-        this.microlocation = microlocation;
-    }
 
     public String generateSql() {
-        String query_normal = "INSERT INTO %s VALUES ('%d', '%s', '%s','%s', '%s', '%s', '%s', '%s', '%d', '%s', '%s', '%d');";
-        String query = String.format(query_normal, DbContract.Sessions.TABLE_NAME, id, title, subtitle, summary, description, startTime, endTime, type, track, speakers, level, microlocation);
+        String query_normal = "INSERT INTO %s VALUES ('%d', '%s', '%s','%s', '%s', '%s', '%s', '%s', '%d', '%s', '%d');";
+        String query = String.format(query_normal, DbContract.Sessions.TABLE_NAME, id, title, subtitle, summary, description, startTime, endTime, type, track, level, microlocations);
         return query;
     }
 
+    public String escapeChar(String string) {
+        return string.replaceAll("'", "''");
```

Comment:
Yes it will but I have used it only once due to error's I was facing wherever we had an inverted comma.